src/nvim/msgpack_rpc/channel.c

```diff
@@ -715,6 +729,22 @@ const char *rpc_client_name(Channel *chan)
   return NULL;
 }
 
+const char *rpc_client_type(Channel *chan)
+{
+  if (!chan->is_rpc) {
+    return NULL;
```

Comment:
I'll make it `"remote"` to avoid Segmantation fault.

Reply:
? the caller can check for NULL.

Reply:
Yes. I simply thought that `type` need not be NULL since the help says that the default value of `type` is `remote`.